tests/amqpvalue_ut/amqpvalue_ut.c

```diff
@@ -210,7 +210,7 @@ TEST_FUNCTION(amqpvalue_create_null_succeeds)
 {
     // arrange
 	AMQP_VALUE result;
-    EXPECTED_CALL(gballoc_malloc(IGNORED_NUM_ARG));
+    STRICT_EXPECTED_CALL(gballoc_malloc(IGNORED_NUM_ARG));
```

Comment:
Looks like most of the changes were for calls that were already ignoring the arguments.
Not sure what is the point of the changes, are you planning on deprecating EXPECTED_CALL(...) ?